mava/wrappers/jumanji.py

```diff
@@ -217,3 +220,128 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             )
 
         return spec
+
+
+class MultiCVRPWrapper(Wrapper):
```

Comment:
Inherit from the `MultiAgentWrapper`

```suggestion
class MultiCVRPWrapper(MultiAgentWrapper):
```